packages/flutter/lib/src/widgets/text_editing_intents.dart

```diff
@@ -16,6 +16,24 @@ class DoNothingAndStopPropagationTextIntent extends Intent {
   const DoNothingAndStopPropagationTextIntent();
 }
 
+/// An [Intent] to send the event straight to the engine when the receiver
+/// is editable text.
+///
+/// When the receiver is not an editable text, then the `fallbackIntent` will
+/// be invoked.
+///
+/// See also:
+///
+///   * [DefaultTextEditingShortcuts], which triggers this [Intent].
+class DoNothingAndStopPropagationEditableTextIntent extends Intent {
```

Comment:
instead of introducing a new intent type, can we handle this in EditableText? When the field is editable on macos, EditableText stops the propagation of the intent and let the embedder handle it, otherwise it handles the input as usual?

Users will still be able to customize by overriding the action.

Reply:
I don't think we can solely handle this in `EditableText` because when `EditableText` receives the `DoNothingAndStopPropagationTextIntent` it has no way of knowing what `ShortcutActivator` activated it so we can't derive some `Intent` for it when the field is editable on macOS.

We would need to change the default `Shortcut -> Intent` mapping to reflect the `SelectableText` one and then in `EditableText` conditionally for each `Intent` choose not to handle them when the field is editable on macOS.

Reply:
Changing the default key binding to the fallback intents, and let editable text handle the MacOS selectors is what I meant.